client/src/components/ReceiveRole.svelte

```diff
@@ -49,7 +69,7 @@
     color: #1976d2;
   }
 
-  .catfish {
+  :global(.catfish) {
```

Comment:
are you sure this should be global?

Reply:
I was using this class in different components so I thought so

Reply:
global does not load it globally it only applies it globally so that would only be guaranteed to work if ReceiveRole is always loaded at the same time as anywhere else that uses it